po-refresh

```diff
@@ -168,19 +169,28 @@ def run(context, verbose=False, **kwargs):
             else:
                 execute("git", "checkout", "--", name)
 
-    # Create a pull request from these changes
     branch = task.branch(context, "po: Update from Fedora Weblate", pathspec="po/",
                          branch=local_branch, **kwargs)
 
-    if branch:
-        task.pull(branch, **kwargs)
-    elif local_branch:
+    if local_branch and not branch:  # We only introduced/dropped language but otherwise no updates
         if kwargs["issue"]:
             clean = "https://github.com/{0}".format(task.find_our_fork(user))
             task.comment_done(kwargs["issue"], "po-refresh", clean, local_branch, context)
 
         task.push_branch(user, local_branch)
-        task.pull("{0}:{1}".format(user, local_branch), **kwargs)
+        branch = "{0}:{1}".format(user, local_branch)
+
+    # Create a pull request from these changes
+    pull = task.pull(branch, labels=['bot', 'no-test'], run_tests=False, **kwargs)
+
+    # Trigger this pull request
+    api = github.GitHub()
+    head = pull["head"]["sha"]
+
+    triggers = testmap.tests_for_po_refresh(api.repo)
```

Comment:
To avoid having to maintain another map, how about this: *if* TEST_OS_DEFAULT is in the existing map, then only run this, otherwise run all contexts (i.e. don't set a no-test label)

Reply:
That assumes that every project that uses default context also wants just that triggered on po refreshes. I don't want to do this assumption.  I think we want that refresh to be treated as any other PR, unless we tell differently - thus the map.

Reply:
OK, fair enough. We can change that in the future if we want to.